jiant/proj/main/modeling/taskmodels.py

```diff
@@ -192,6 +194,7 @@ def forward(self, batch, task, tokenizer, compute_loss: bool = False):
             input_ids=masked_batch.masked_input_ids,
             segment_ids=masked_batch.segment_ids,
             input_mask=masked_batch.input_mask,
+            batch=batch,
```

Comment:
might as well just pass in `batch`

Reply:
True, this can be switched to `get_output_from_encoder_and_batch`.